packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -875,41 +484,17 @@ class FlutterDriver {
   ///     ]
   ///
   /// [getFlagList]: https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#getflaglist
-  Future<List<Map<String, dynamic>>> getVmFlags() async {
-    await _restorePeerConnectionIfNeeded();
-    final Map<String, dynamic> result = await _peer.sendRequest('getFlagList');
-    return result != null
-        ? result['flags'].cast<Map<String,dynamic>>()
-        : const <Map<String, dynamic>>[];
-  }
+  Future<List<Map<String, dynamic>>> getVmFlags();
```

Comment:
it would be good to put all the abstract methods in one place so that someone implementing this interface just has one block of code to reference